dsl/common/src/main/kotlin/io/mockk/API.kt

```diff
@@ -2254,6 +2264,13 @@ object MockKCancellationRegistry {
             map[key] = newCancellation
         }
 
+        fun putIfNotThere(key: Any, newCancellation: MockKCancellation) {
```

Comment:
I think you can achieve the same behavior by using the standard library's [putIfAbsent](https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#putIfAbsent-K-V-)

Reply:
This map is a kotlin map, which does not offer `putIfAbsent` like Java 8 map. Is there something similar in kotlin's map?

Reply:
No, you're right.